### PR TITLE
CB-15404 Introduce Cloudera Runtime 7.2.15

### PR DESCRIPTION
**Pull request by pkedvessy**
See detailed description in the commit message.

### CODE REVIEW

File: datalake/src/main/resources/application.yml [hunk 1]
```diff
@@ -111,8 +111,8 @@ datalake:
       ssl: false
   runtimes:
     default: "7.2.13"
-    advertised: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.14"
-    supported: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.14"
+    advertised: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.147.2.15"
```
Comment (horadla23) [inline datalake/src/main/resources/application.yml:114]: `,` is missing

**Review by horadla23 — CHANGES_REQUESTED** [review]

**Comment by horadla23** [comment]
is it required for 7.2.15 to be in 2.51? we are about to release it, I guess it should go only in master

**Comment by lnardai** [comment]
Yes, we've discussed this with @TheTinkerDad and that this will only be merged, after 2.51 is released.

File: datalake/src/main/resources/application.yml (hunk 1 above)
Comment (pkedvessy) [reply datalake/src/main/resources/application.yml:114]: Ahh, thanks.

**Review by horadla23 — APPROVED** [review]

**Comment by pkedvessy** [comment]
Support for 7.2.15 has been moved to CB 2.52. It should be available on mow-dev by Monday (02.18) or so.

**Pull request closed without merging**